devops-infra/def/network/.htaccess

```diff
@@ -0,0 +1,49 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+# Rewrite engine setup
+RewriteEngine On
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/ontology/network/index-en.html [R=302,L]
+
+# Rewrite rule to serve JSON-LD content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/ontology/network/ontology.json [R=302,L]
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/ontology/network/ontology.xml [R=302,L]
+
+# Rewrite rule to serve N-Triples content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/n-triples
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/ontology/network/ontology.nt [R=302,L]
+
+# Rewrite rule to serve TTL content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/turtle 
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/ontology/network/ontology.ttl [R=302,L]
+
+RewriteCond %{HTTP_ACCEPT} .+
+RewriteRule ^network$ https://oeg-upm.github.io/devops-infra/ontology/network/406.html [R=302,L]
+RewriteRule ^software$ https://oeg-upm.github.io/devops-infra/ontology/software/406.html [R=302,L]
+RewriteRule ^database$ https://oeg-upm.github.io/devops-infra/ontology/database/406.html [R=302,L]
+RewriteRule ^hardware$ https://oeg-upm.github.io/devops-infra/ontology/hardware/406.html [R=302,L]
+RewriteRule ^certificate$ https://oeg-upm.github.io/devops-infra/ontology/certificate/406.html [R=302,L]
```

Comment:
I know this has been merged, but just noting these rules differ from your other files.